Source/HBIOS/nabukb.asm

```diff
@@ -0,0 +1,86 @@
+;======================================================================
+;	NABU KEYBOARD DRIVER
+;
+;	CREATED BY: LES BIRD
+;
+;======================================================================
+;
+; KBPORT EQU	$90
+;
+; POLL FOR INPUT

```

Comment:
This commented out code example is from the NABU ROM which I used as a reference